### PR TITLE
implemented User class with preliminary methods

### PR DESCRIPTION
**Pull request by rpeikes**

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -54,7 +54,49 @@ def delete(self, ID):
         return '', 204
 
 
+class User(Resource):
+    @marshal_with(resource_fields_item)
```
Comment (TzipHoro) [inline main.py]: `@marshal_with(resource_fields_user)`

File: main.py [hunk 2]
```diff
@@ -54,7 +54,49 @@ def delete(self, ID):
...
+    def patch(self, ID):
+        args = user_update_args.parse_args()
+        result = Item_Model.query.filter_by(user_id=ID).first()
+        if not result:
+            abort(404, message="User Id does not exist...")
+
+        if args['user_id']:
+            result.user_id = args['user_id']
+        if args['pref_name']:
+            result.title = args['pref_name']
```
Comment (TzipHoro) [inline main.py]: I think this should be `result.pref_name = args['pref_name']`

File: main.py (hunk 1 above)
Comment (rpeikes) [reply main.py]: lol good call. thanks!

**Review by TzipHoro — CHANGES_REQUESTED** [review]
It looks like there are some minor copy/paste errors...

**Comment by rpeikes** [comment]
fixed

**Review by TzipHoro — APPROVED** [review]

**Pull request merged**